scripts/ci/run_perftest.sh

```diff
@@ -43,7 +47,8 @@ cat << EOF > /etc/telegraf/telegraf.conf
   json_time_format = "unix"
   tag_keys = [
     "i_type",
-    "query_format"
+    "query_format",
+    "is_metaquery"
```

Comment:
Instead of a boolean here, should we have a string tag with some constant set of values? One of which would be "metaquery", but others could be "window-aggregate", "group-aggregate", etc

Reply:
I think it'd also be useful to have a "metaquery_type" tag with value "field-keys" or "field-values", to support drilling down even further. I plan to do something similar and have an "aggregate_type" tag with value count/min/max/etc.

Reply:
Yeah that seems like a good idea as we start adding more tests. I'll make new tags with keys `use_case` and `query_type` to match the syntax of the benchmarking tools.